src/wagtail_factories/factories.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 import factory
+from factory.django import DjangoModelFactory
```

Comment:
Suggest this should probably try/except for `factory.DjangoModelFactory` to preserve compatibility with Factory Boy 2.x.

```python
try:
    from factory.django import DjangoModelFactory
except ImportError:
    from factory import DjangoModelFactory
```

This might help the PR pass tox tests also.